src/test/resources/com/github/dockerjava/api/command/inspectContainerResponse_full_healthcheck.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
i made new apis for json dumps, there is clear structure with versions and places from what they come. So this also need to be moved.